docker/docker-compose.test.yml

```
@@ -0,0 +1,12 @@
+services:
+  cypress:
+    image: cypress/included
+    environment:
+      CYPRESS_BASE_URL: http://ui:3000
+      OASTUB_URL: http://oastub:3000/
```

Prepending `CYPRESS_` to a environment variable will make it override the respective value in the cypress.config.ts
```suggestion
      CYPRESS_OASTUB_URL: http://oastub:3000
```